inquirer/themes.py

```diff
@@ -69,47 +69,105 @@ def load_theme_from_dict(dict_theme):
+        self.opening_prompt_color = term.bright_black
+
+
+class Checkbox(ThemeItem):
+    def __init__(self):
+        super(Checkbox, self).__init__()
+        self.selection_color = term.blue
+        self.selection_icon = ">"
+        self.selected_icon = "X"
+        self.selected_color = term.yellow + term.bold
+        self.unselected_color = term.normal
+        self.unselected_icon = "o"
+
+
+class List(ThemeItem):
+    def __init__(self):
+        super(List, self).__init__()
+        self.selection_color = term.blue
+        self.selection_cursor = ">"
+        self.unselected_color = term.normal
+
+
+class Theme(object):
+    def __init__(self):
+        super(Theme, self).__init__()
+        self.question = Question()
+        self.text = Text()
+        self.path = Path()
+        self.password = Password()
+        self.confirm = Confirm()
+        self.editor = Editor()
+        self.checkbox = Checkbox()
+        self.list = List()
 
 
 class Default(Theme):
     def __init__(self):
         super(Default, self).__init__()
-        self.Question.mark_color = term.yellow
-        self.Question.brackets_color = term.normal
-        self.Question.default_color = term.normal
-        self.Editor.opening_prompt_color = term.bright_black
-        self.Checkbox.selection_color = term.blue
-        self.Checkbox.selection_icon = ">"
-        self.Checkbox.selected_icon = "X"
-        self.Checkbox.selected_color = term.yellow + term.bold
-        self.Checkbox.unselected_color = term.normal
-        self.Checkbox.unselected_icon = "o"
-        self.List.selection_color = term.blue
-        self.List.selection_cursor = ">"
-        self.List.unselected_color = term.normal
 
 
 class GreenPassion(Theme):
     def __init__(self):
         super(GreenPassion, self).__init__()
-        self.Question.mark_color = term.yellow
-        self.Question.brackets_color = term.bright_green
-        self.Question.default_color = term.yellow
-        self.Checkbox.selection_color = term.bold_black_on_bright_green
-        self.Checkbox.selection_icon = "❯"
-        self.Checkbox.selected_icon = "◉"
-        self.Checkbox.selected_color = term.green
-        self.Checkbox.unselected_color = term.normal
-        self.Checkbox.unselected_icon = "◯"
-        self.List.selection_color = term.bold_black_on_bright_green
-        self.List.selection_cursor = "❯"
-        self.List.unselected_color = term.normal
+        self.question.brackets_color = term.bright_green
+        self.question.default_color = term.yellow
+        self.checkbox.selection_color = term.bold_black_on_bright_green
+        self.checkbox.selection_icon = "❯"
+        self.checkbox.selected_icon = "◉"
+        self.checkbox.selected_color = term.green
+        self.checkbox.unselected_color = term.normal
+        self.checkbox.unselected_icon = "◯"
+        self.list.selection_color = term.bold_black_on_bright_green
+        self.list.selection_cursor = "❯"
```

Comment:
same as before.. I don't know a good name maybe `choices` instead of `list`?

Reply:
Everytime i use the library i end up importing it as iList...